sideboard/lib/sa/__init__.py

```diff
@@ -261,7 +282,7 @@ def resolve_model(cls, name):
                     return subclasses[singular]
 
             if name.lower().endswith('ies'):
-                singular = name[:-3] + 'sy'  # TODO: sy looks like a typo, and we need to either make this better or get rid of it
+                singular = name[:-3] + 'y'
```

Comment:
TODOs like this should be banned. Either fix it immediately or create a ticket for it.

This is unrelated to issue #72, but I fixed it now while I'm looking at it so I don't forget about it.